<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookingsScreen</class>
 <widget class="QDialog" name="BookingsScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>420</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(29, 13, 32); </string>
  </property>
  <widget class="QTableWidget" name="bookingsTableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>550</width>
     <height>350</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>550</width>
     <height>350</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>550</width>
     <height>350</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(29, 13, 32);
color: rgb(73, 236, 253);
gridline-color: rgb(73, 236, 253);
border: 1px solid rgb(73, 236, 253);

selection-background-color: rgb(73, 236, 253);
selection-color: rgb(29, 13, 32);</string>
   </property>
   <column>
    <property name="text">
     <string>Имя</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Дата</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Время</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Устройства</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>30</y>
     <width>143</width>
     <height>351</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item alignment="Qt::AlignmentFlag::AlignHCenter">
     <widget class="QDateEdit" name="searchDateEdit">
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>110</width>
        <height>22</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(73, 236, 253);</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignmentFlag::AlignHCenter">
     <widget class="QLineEdit" name="numberLineEdit">
      <property name="minimumSize">
       <size>
        <width>141</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>141</width>
        <height>21</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(29, 13, 32);
color: rgb(73, 236, 253);
border: 1px solid rgb(73, 236, 253);</string>
      </property>
      <property name="placeholderText">
       <string>Введите номер ПК</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignmentFlag::AlignHCenter">
     <widget class="QLineEdit" name="nameLineEdit">
      <property name="minimumSize">
       <size>
        <width>141</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>141</width>
        <height>21</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(29, 13, 32);
color: rgb(73, 236, 253);
border: 1px solid rgb(73, 236, 253);</string>
      </property>
      <property name="placeholderText">
       <string>Введите Имя</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignmentFlag::AlignHCenter">
     <widget class="QPushButton" name="searchButton">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>32</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(73, 236, 253);
color: rgb(29, 13, 32);
border-radius: 5px;</string>
      </property>
      <property name="text">
       <string>Поиск</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignmentFlag::AlignHCenter">
     <widget class="QPushButton" name="deleteButton">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>32</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(73, 236, 253);
color: rgb(29, 13, 32);
border-radius: 5px;</string>
      </property>
      <property name="text">
       <string>Удалить бронь</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
